src/Management/src/TracingBase/TracingBaseServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,84 @@
+    {
+        /// <summary>
+        /// Configure distributed tracing via OpenTelemetry with HttpClient Instrumentation.
+        /// </summary>
+        /// <param name="services"><see cref="IServiceCollection" /></param>
+        /// <returns><see cref="IServiceCollection"/> configured for distributed tracing.</returns>
+        public static IServiceCollection AddDistributedTracing(this IServiceCollection services) => services.AddDistributedTracing(null);
+
+        /// <summary>
+        /// Configure distributed tracing via OpenTelemetry with HttpClient Instrumentation.
+        /// </summary>
+        /// <param name="services"><see cref="IServiceCollection" /></param>
+        /// <param name="action">Customize the <see cref="TracerProviderBuilder" />.</param>
+        /// <returns><see cref="IServiceCollection"/> configured for distributed tracing.</returns>
+        public static IServiceCollection AddDistributedTracing(this IServiceCollection services, Action<TracerProviderBuilder> action)
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.AddOptions();
+            services.RegisterDefaultApplicationInstanceInfo();
+            services.TryAddSingleton<ITracingOptions>((serviceProvider) => new TracingOptions(serviceProvider.GetRequiredService<IApplicationInstanceInfo>(), serviceProvider.GetRequiredService<IConfiguration>()));
+            services.TryAddSingleton<IDynamicMessageProcessor, TracingLogProcessor>();
+
+            services.AddOpenTelemetryTracing(builder =>
+            {
+                builder.Configure((serviceProvider, deferredBuilder) =>
+                {
+                    var appName = serviceProvider.GetRequiredService<IApplicationInstanceInfo>().ApplicationNameInContext(SteeltoeComponent.Management, TracingOptions.CONFIG_PREFIX + ":name");
+                    var traceOpts = serviceProvider.GetRequiredService<ITracingOptions>();
+                    deferredBuilder.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(appName));
+                    deferredBuilder.AddHttpClientInstrumentation(options =>
+                        {
+                            var pathMatcher = new Regex(traceOpts.EgressIgnorePattern);
+                            options.Filter += req => !pathMatcher.IsMatch(req.RequestUri.PathAndQuery);
+                        });
+
+                    if (traceOpts.PropagationType.Equals("B3", StringComparison.InvariantCultureIgnoreCase))
+                    {
+                        var propagators = new List<TextMapPropagator> { new B3Propagator(traceOpts.SingleB3Header), new BaggagePropagator() };
+                        Sdk.SetDefaultTextMapPropagator(new CompositeTextMapPropagator(propagators));
+                    }
+
+                    // To Discuss: Should these remain Steeltoe options or downstream set directly?
```

Comment:
Samplers can be set via the exposed `TracerProviderBuilder`, should we keep the config-based option?